rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -95,5 +95,105 @@
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>IMPLEMENTS_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.Files{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.FileStore{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.FileSystem{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.FileSystems{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.Path{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
+                  </javaclass>
+                  <javaclass references="java.nio.file.Paths{*}">
+                      <location>IMPORT</location>
+                      <location>INHERITANCE</location>
+                      <location>RETURN_TYPE</location>
+                      <location>VARIABLE_DECLARATION</location>
+                      <location>CONSTRUCTOR_CALL</location>
+                      <location>METHOD_CALL</location>
```

Comment:
No, it's not important to exclude some of them: it's the result of merging other rules together.
I delete the locations.